scripts/gathering/generateLandOwner.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * How to use:
+ *  - yarn execute <NETWORK> ./scripts/gathering/generateLandOwner.ts <blockNumber>
+ */
+import 'dotenv/config';
+import 'isomorphic-unfetch';
+import {createClient} from '@urql/core';
+
+import ObjectsToCsv from 'objects-to-csv';
+
+async function query<T>(
+  queryString: string,
+  field: string,
+  variables: Record<string, unknown>
+): Promise<T[]> {
+  const first = 1000;
+  let entries: T[] = [];
+  // Max value for x coordinate is 407
```

Comment:
407 or 408?
you use gte
```x_gte <= 408```

Reply:
The value of x and y ranges from 0 to 407 which aligns with the 408*408 grid